src/BlogServiceProvider.php

```diff
@@ -32,6 +35,25 @@ public function configureTipoffPackage(TipoffPackage $package): void
                 \DrewRoberts\Blog\Nova\Series::class,
                 \DrewRoberts\Blog\Nova\Topic::class,
             ])
+            ->hasWebRoute('web')
+            ->hasViews()
             ->name('blog');
     }
+
+    public function bootingPackage()
+    {
+        parent::bootingPackage();
+
+        // Must happen AFTER SubstituteBindings middleware has been applied
+        app(Kernel::class)->appendToMiddlewarePriority(ResolvePage::class);
+
+        Route::model('page', Page::class);
```

Comment:
@pdbreen I add the binding model to the boot package to use page in the routes as  `Route::get('{page}', PageController::class)`, but is not working, I receive in the _invoke method in the controller `Page` empty, what I am missing?